tensorflow/core/common_runtime/graph_constructor_fuzz.cc

```diff
@@ -28,5 +29,40 @@ void FuzzImportGraphDef(const GraphDef& graph_def) {
 }
 FUZZ_TEST(GraphDefFuzz, FuzzImportGraphDef);
 
+void FuzzGraphEndToEnd1(const GraphDef& graph_def) {
```

Comment:
nit: do we need that one at the end? If there is a duplicate, could you change the name to describe the difference?